janitor/finance.py

```diff
@@ -395,6 +402,237 @@ def _convert_currency(
     return rate
 
 
+@pf.register_dataframe_method
+@deprecated_alias(col_name="column_name", type="cleaning_style")
+@refactored_function(
```

Comment:
Hi @dendrondal, you don't need to import and implement `@refactored_function(...)` here.

It should go in `__init__.py`, and you can follow the pattern that `get_features_targets` uses:

```python
from .finance import currency_column_to_numeric as _currency_column_to_numeric

@refactored_function(
    "currency_column_to_numeric() has moved. Please use finance.currency_column_to_numeric()."
)
def currency_column_to_numeric(*args, **kwargs):
    return _currency_column_to_numeric(*args, **kwargs)

```

If I recall, that should be the last thing to push this PR through!

Reply:
@hectormz, thank you for conducting the review! I'm also learning from you how this pattern can be used :smile:. 

@dendrondal - thanks for handling this! Once you're done with that, don't worry about the conflict on CHANGELOG. I will take care of that through the web editor.

Reply:
👍 Perhaps I should write a small section in `CONTRIBUTING.rst` on how and why to refactor functions and argument names with the wrappers we've implemented. Maybe even a list item in the PR template whether or not someone has moved a function/renamed arguments. Might be overkill though...

Reply:
This should do the trick! I think adding a subsection to CONTRIBUTING.rst is a good idea, but it seems like such a rare occurrence that I don't know if it needs to be in the PR template.